modules/Hoot/ui/conflicts/merge.js

```diff
@@ -342,4 +340,34 @@ export default class Merge {
 
         Hoot.context.background().updateArrowLayer( gj );
     }
+
+    checkMergeButton() {
+        let features = _clone( this.data.currentFeatures ),
+            toFeature = Hoot.context.hasEntity( features[0].id ),
+            fromFeature = Hoot.context.hasEntity( features[1].id ),
+            relation = this.instance.graphSync.getCurrentRelation();
+
+        if ( toFeature && fromFeature) {
+            this.isPoiPoly = (toFeature.type === 'node' && fromFeature.type === 'way')
+                || (toFeature.type === 'way' && fromFeature.type === 'node');
```

Comment:
We need to define POI->Poly merge this way (instead of using `hoot:review:type` because after a merge, other review relations will be updated to refer to the merged element.  After a poi->poly merge the referred element may no long be a POI and may be two polys but the review type remains unchanged.  This may change when we implement this long ago ticket #1003 to allow poly->poly merges.